style.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Poppins:wght@200;300;400;500;600;700;800&display=swap');
+
+body{
+  position: relative;
+
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+
+  background: radial-gradient(ellipse at bottom, #022227 0%, #000000 100%);
+}
+
+h1{
+  position: absolute;
+  top: 13vh;
+  left: 50%;
+
+  margin: 0;
+
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 900;
+  font-size: 30vh;
+  letter-spacing: 1vw;
+
+  color: white;
+  background:url('https://thumbs.dreamstime.com/b/tropical-flower-pattern-seamless-flowers-blossom-flowers-nature-background-vector-illustration-61142748.jpg');
+  -webkit-box-reflect: below -0.45em linear-gradient(to bottom, rgba(0,0,0,0.0), rgba(0,0,0,0.4));
+  background-size: auto 200%;
+  background-clip: text;
+  -webkit-text-fill-color: transparent;
```

Comment:
I think you mistakenly write -webkit- here because in line number 32 vendor prefix is there so it should be without the vendor prefix...

Reply:
I have not added it earlier but it was showing me yellow underline in VS Code so I have added prefix.